### ISSUE TITLE
[Feature] Slider to allow more floors

### ISSUE THREAD

**Issue by maksw2** [SuperMartijn642/MovingElevators#233]
**Is your feature request related to a problem? Please describe.**
I am building a WTC replica, but the amount of floors make it hard to navigate through all the levels (60 floors).

**Describe the solution you'd like**
A slider to move through the list.
Something like that:
![image](https://github.com/user-attachments/assets/ad66f6b5-e0f2-4fb3-a7f2-ec254bf1389b)

**Comment by ClanSingapore** [comment]
yeah we need this feature, I'm trying to build a 24 floor apartment block but can only settle with 15 floors so far, hope they can add this for 1.21.5
